test/notificationSpec.js

```diff
@@ -639,124 +655,9 @@ describe('Notifications', function () {
         .end()
     })
 
-    it('should execute a payment where there are multiple debits from its account in the destination transfer', function * () {
```

Comment:
these tests should be replaced with one that checks that the connector won't accept these cases